### DISCUSSION TITLE
Expedition 13

### DISCUSSION THREAD

**Discussion by jcjjunior** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#4] [General]
Hello
I have a problem. I followed the entire procedure as recommended in the video and as soon as the expedition starts... it says that expedition 13 has been completed. When I open it and see if the missions to get the rewards have been completed, everything is shown as not completed. Could you help me? Thank you.
Sorry for my English. :/

**Comment by cwmonkey** [comment]
If you are using Steam, you need to put it in "offline" mode BEFORE opening the game. For other platforms, make sure they are similarly offline.

**Reply by jcjjunior** [reply]
Yes, I am using it on Steam. As I said, I followed all the steps in the Dread Captain James tutorial. I disabled the internet on Steam and did the same test by turning off the internet on the computer. Thank you for your super quick attention. Thank you.

![NMS](https://github.com/leodium/leodium-ReLive-NoMansSky-Expeditions-Offline/assets/143941316/e8f9b5a5-ba02-41d1-bcdb-0bbe52a9b234)

**Reply by cwmonkey** [reply]
Hm, I'm not sure what the issue is. If you want to restart expedition 13 offline you could do the following:
1. With the game closed, replace the `SEASON_DATA_CACHE.JSON` with season 1's offline file
2. Start the game and load your save
3. You will get the message saying expedition 13 is over, just accept all the prompts
4. Go to the expedition terminal and start expedition 1
5. Save your game and close nms
6. Replace the `SEASON_DATA_CACHE.JSON` file with the expedition 13 file
7. Start the game and load your save, you should get the messages the expedition has ended
8. Go to the expedition terminal and start expedition 13 again
